tests/edjs.test.ts

```diff
@@ -1,44 +1,44 @@
-import parser from "../src/main";
-import data from "./data.json";
-import corruptData from "./shittyData.json";
+import parser from '../src/main';
+import data from './data.json';
+import corruptData from './shittyData.json';
 
-test("All Methods are defined and available", () => {
+test('All Methods are defined and available', () => {
   const edjsParser = parser();
 
-  expect(edjsParser["parse"]).toBeDefined();
-  expect(edjsParser["parseBlock"]).toBeDefined();
+  expect(edjsParser['parse']).toBeDefined();
+  expect(edjsParser['parseBlock']).toBeDefined();
 });
 
-describe("Testing Parser Transformer Functions", () => {
+describe('Testing Parser Transformer Functions', () => {
   const edjsParser = parser();
 
-  test("Code Works", () => {
+  test('Code Works', () => {
     const block = {};
   });
 
-  test("Delimiter Works", () => {
+  test('Delimiter Works', () => {
     const delimiterBlock = {
-      type: "delimiter",
+      type: 'delimiter',
       data: {},
     };
 
     const html = edjsParser.parseBlock(delimiterBlock);
-    expect(html).toBe("<br/>");
+    expect(html).toBe('<br/>');
   });
 
-  test("Header Works", () => {
+  test('Header Works', () => {
```

Comment:
@Luzefiru Thanks for the quick PR. Can you please update the PR to use double quotes, your formatter has converted all the code to single quotes.
I'd recommend if you could add prettier config. But just updating these quotes would be good enough.

Reply:
A prettier config would be good for the longevity of the project. I'll make it tomorrow when I have time.

Thank you for your feedback!